2017-bae-europe/eu_future/buff_but_sink.xml

```diff
@@ -0,0 +1,5827 @@
+        <fuel_incommods>  <val>uox</val>                   <val>mox</val>                    </fuel_incommods>
+        <fuel_outcommods> <val>french_uox_waste</val>             <val>mox_waste</val>              </fuel_outcommods>
+        <fuel_prefs>      <val>1.0</val>                   <val>2.0</val>                    </fuel_prefs>
+        <cycle_time>18</cycle_time>
+        <refuel_time>2</refuel_time>
+        <assem_size>523.4</assem_size>
+        <n_assem_core>176</n_assem_core>
+        <n_assem_batch>59</n_assem_batch>
+        <power_cap>910</power_cap>
+      </Reactor>
+    </config>
+  </facility>
+
+ <facility>
+    <!-- France -->
+    <name>CRUAS-1</name>
+    <config>
+      <Reactor>
+        <fuel_inrecipes>  <val>uox_fuel_recipe</val>       <val>mox_fuel_recipe</val>        </fuel_inrecipes>
+        <fuel_outrecipes> <val>uox_used_fuel_recipe</val>  <val>mox_used_fuel_recipe</val>   </fuel_outrecipes>
+        <fuel_incommods>  <val>uox</val>                   <val>mox</val>                    </fuel_incommods>
+        <fuel_outcommods> <val>french_uox_waste</val>             <val>mox_waste</val>              </fuel_outcommods>
+        <fuel_prefs>      <val>1.0</val>                   <val>2.0</val>                    </fuel_prefs>
+        <cycle_time>18</cycle_time>
+        <refuel_time>2</refuel_time>
+        <assem_size>523.4</assem_size>
+        <n_assem_core>177</n_assem_core>
+        <n_assem_batch>59</n_assem_batch>
+        <power_cap>915</power_cap>
+      </Reactor>
+    </config>
+  </facility>
+
+ <facility>
+    <!-- France -->
+    <name>CRUAS-2</name>
+    <config>
+      <Reactor>
+        <fuel_inrecipes>  <val>uox_fuel_recipe</val>       <val>mox_fuel_recipe</val>        </fuel_inrecipes>
+        <fuel_outrecipes> <val>uox_used_fuel_recipe</val>  <val>mox_used_fuel_recipe</val>   </fuel_outrecipes>
+        <fuel_incommods>  <val>uox</val>                   <val>mox</val>                    </fuel_incommods>
+        <fuel_outcommods> <val>french_uox_waste</val>             <val>mox_waste</val>              </fuel_outcommods>
+        <fuel_prefs>      <val>1.0</val>                   <val>2.0</val>                    </fuel_prefs>
+        <cycle_time>18</cycle_time>
+        <refuel_time>2</refuel_time>
+        <assem_size>523.4</assem_size>
+        <n_assem_core>177</n_assem_core>
+        <n_assem_batch>59</n_assem_batch>
+        <power_cap>915</power_cap>
+      </Reactor>
+    </config>
+  </facility>
+
+ <facility>
+    <!-- France -->
+    <name>ST. ALBAN-1</name>
+    <config>
+      <Reactor>
+        <fuel_inrecipes>  <val>uox_fuel_recipe</val>       <val>mox_fuel_recipe</val>        </fuel_inrecipes>
+        <fuel_outrecipes> <val>uox_used_fuel_recipe</val>  <val>mox_used_fuel_recipe</val>   </fuel_outrecipes>
+        <fuel_incommods>  <val>uox</val>                   <val>mox</val>                
```

Comment:
What would be the most concise way to point to the VISION recipe excel file? ( Since the github repository is private,)